Sensors.cpp

```diff
@@ -3,6 +3,20 @@
 // Humidity sensor
 Adafruit_AM2320 am2320 = Adafruit_AM2320();
 
+//BME280
+ #define SEALEVELPRESSURE_HPA (1013.25)
+ Adafruit_BME280 bme;           // BME280 I2C
+//AHT10
+ Adafruit_AHTX0 aht;
+ Adafruit_Sensor *aht_humidity, *aht_temp;
+//SHT31
+ Adafruit_SHT31 sht31 = Adafruit_SHT31();
+//DHT22
+ #define DHTTYPE DHT22         // DHT 22 (AM2302)
+ #define DHTPIN 15
```

Comment:
maybe we need pass it on the init of the library, because otherwise the user is forced only to this pin. The other humidity sensors could using the same pin?

Reply:
The DHT22 use its own protocol serial 1wire, the others are i2c. The pin have to be different for the use of the oled that use i2c

Reply:
ok, then we need a setter  or parameter on init for it. Maybe somethng like this:

```C
void Sensors::init(int pms_type, int pms_rx, int pms_tx, int dhtpin)
``` 
Please send that in a new commit.